deploy.py

```diff
@@ -29,6 +29,24 @@ def parse_clusters():
     ]
 
 
+def find_cluster(cluster_name, clusters):
```

Comment:
Since these are called only in one place right now, I think we should inline them instead of splitting it out into a function. We can refactor it later if needed.

Reply:
If you're feeling fancy, you can also use https://stackoverflow.com/a/34894322 instead of a for loop :)

Reply:
ty <3